src/Video/Lighting/Lighting.cpp

```diff
@@ -79,15 +79,14 @@ void Lighting::Render(const glm::mat4& inverseProjectionMatrix, RenderSurface* r
     //glUniform1i(shaderProgram->GetUniformLocation("textureMetallic"), 2);
     //glUniform1i(shaderProgram->GetUniformLocation("textureRougness"), 3);
     //glUniform1i(shaderProgram->GetUniformLocation("tDepth"), 4);
-    //
+
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') comma should be followed by whitespace [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT009)